### PR TITLE
Update RS of MS-OXWSITEMID according to the document of v20181001

### PR DESCRIPTION
**Pull request by huangxiaochen1**

### CODE REVIEW

**Review by v-yahan — COMMENTED** [review]
1. Please add Verification Comment for MS-OXWSCORE_R2400: Verified by derived requirement: MS-OXWSCORE_R2919.
2. MS-OXWSCONRE_R1087 is non-testable and it verified in MS-OXWSCORE_R2935, so it also need to add verification Comment.

File: ExchangeWebServices/Source/MS-OXWSCORE/TestSuite/MS-OXWSCORE_RequirementSpecification.xml [hunk 1]
```diff
@@ -1323,6 +1330,15 @@
 		<ns1:IsNormative>Normative</ns1:IsNormative>
 		<ns1:Verification>Unverified</ns1:Verification>
 	</ns1:Requirement>
+	<ns1:Requirement>
+		<ns1:REQ_ID>MS-OXWSCORE_R2309</ns1:REQ_ID>
+		<ns1:Doc_Sect>2.2.4.24</ns1:Doc_Sect>
+		<ns1:Description>[In t:ItemType Complex Type] This element [MimeContent] is applicable for all item objects when retrieving MIME content.&lt;52&gt;.</ns1:Description>
+		<ns1:Behavior>Protocol</ns1:Behavior>
+		<ns1:Scope>Both</ns1:Scope>
+		<ns1:IsNormative>Normative</ns1:IsNormative>
```
Comment (FangfangFan) [inline ExchangeWebServices/Source/MS-OXWSCORE/TestSuite/MS-OXWSCORE_RequirementSpecification.xml:1339]: Should derive new RS from endnote <52>

File: ExchangeWebServices/Source/MS-OXWSCORE/TestSuite/MS-OXWSCORE_RequirementSpecification.xml [hunk 2]
```diff
@@ -3059,10 +3066,19 @@
 		<ns1:IsNormative>Normative</ns1:IsNormative>
 		<ns1:Verification>Test Case</ns1:Verification>
 	</ns1:Requirement>
+	<ns1:Requirement>
```
Comment (FangfangFan) [inline ExchangeWebServices/Source/MS-OXWSCORE/TestSuite/MS-OXWSCORE_RequirementSpecification.xml:3069]: it should be derived RS to verify like MS-OXWSCORE_R1730. Please add New RS and Comments

Comment (FangfangFan) [reply ExchangeWebServices/Source/MS-OXWSCORE/TestSuite/MS-OXWSCORE_RequirementSpecification.xml:3069]: or like MS-OXWSCORE_R1732

Comment (FangfangFan) [reply ExchangeWebServices/Source/MS-OXWSCORE/TestSuite/MS-OXWSCORE_RequirementSpecification.xml:3069]: <ns1:REQ_ID>MS-OXWSCORE_R1730</ns1:REQ_ID>
....
<ns1:VerificationComment>Verified by derived requirement: MS-OXWSCORE_R1731.</ns1:VerificationComment>

File: ExchangeWebServices/Source/MS-OXWSCORE/TestSuite/MS-OXWSCORE_RequirementSpecification.xml [hunk 3]
```diff
@@ -3059,10 +3066,19 @@
...
+		<ns1:REQ_ID>MS-OXWSCORE_R2400</ns1:REQ_ID>
+		<ns1:Doc_Sect>2.2.4.24</ns1:Doc_Sect>
+		<ns1:Description>[In t:ItemType Complex Type] [The element "MimeContentUTF8" with type "t:MimeContentUTF8Type (section 2.2.4.29)"] Specifies an instance of the MimeContentUTF8Type complex type that contains the native MIME stream of an object that is represented in UTF-8.&lt;79&gt;</ns1:Description>
+		<ns1:Behavior>Protocol</ns1:Behavior>
+		<ns1:Scope>Both</ns1:Scope>
+		<ns1:IsNormative>Normative</ns1:IsNormative>
+		<ns1:Verification>Non-testable</ns1:Verification>
+	</ns1:Requirement>
 	<ns1:Requirement>
 		<ns1:REQ_ID>MS-OXWSCORE_R2064</ns1:REQ_ID>
```
Comment (FangfangFan) [inline ExchangeWebServices/Source/MS-OXWSCORE/TestSuite/MS-OXWSCORE_RequirementSpecification.xml:3079]: it should be derived RS to verify like MS-OXWSCORE_R1730. Please add New RS and Comments

File: ExchangeWebServices/Source/MS-OXWSCORE/TestSuite/MS-OXWSCORE_RequirementSpecification.xml [hunk 4]
```diff
@@ -3071,7 +3087,7 @@
 	<ns1:Requirement>
 		<ns1:REQ_ID>MS-OXWSCORE_R2065</ns1:REQ_ID>
 		<ns1:Doc_Sect>2.2.4.24</ns1:Doc_Sect>
```
Comment (FangfangFan) [inline ExchangeWebServices/Source/MS-OXWSCORE/TestSuite/MS-OXWSCORE_RequirementSpecification.xml:3089]: it should be derived RS to verify like MS-OXWSCORE_R1730. Please add New RS and Comments

File: ExchangeWebServices/Source/MS-OXWSCORE/TestSuite/MS-OXWSCORE_RequirementSpecification.xml [hunk 5]
```diff
@@ -3080,7 +3096,7 @@
 	<ns1:Requirement>
 		<ns1:REQ_ID>MS-OXWSCORE_R2066</ns1:REQ_ID>
 		<ns1:Doc_Sect>2.2.4.24</ns1:Doc_Sect>
```
Comment (FangfangFan) [inline ExchangeWebServices/Source/MS-OXWSCORE/TestSuite/MS-OXWSCORE_RequirementSpecification.xml:3098]: it should be derived RS to verify like MS-OXWSCORE_R1730. Please add New RS and Comments

File: ExchangeWebServices/Source/MS-OXWSCORE/TestSuite/MS-OXWSCORE_RequirementSpecification.xml [hunk 6]
```diff
@@ -3089,7 +3105,7 @@
 	<ns1:Requirement>
 		<ns1:REQ_ID>MS-OXWSCORE_R2067</ns1:REQ_ID>
 		<ns1:Doc_Sect>2.2.4.24</ns1:Doc_Sect>
```
Comment (FangfangFan) [inline ExchangeWebServices/Source/MS-OXWSCORE/TestSuite/MS-OXWSCORE_RequirementSpecification.xml:3107]: it should be derived RS to verify like MS-OXWSCORE_R1730. Please add New RS and Comments

File: ExchangeWebServices/Source/MS-OXWSCORE/TestSuite/MS-OXWSCORE_RequirementSpecification.xml [hunk 7]
```diff
@@ -3107,7 +3123,7 @@
 	<ns1:Requirement>
 		<ns1:REQ_ID>MS-OXWSCORE_R2069</ns1:REQ_ID>
```
Comment (FangfangFan) [inline ExchangeWebServices/Source/MS-OXWSCORE/TestSuite/MS-OXWSCORE_RequirementSpecification.xml:3124]: it should be derived RS to verify like MS-OXWSCORE_R1730. Please add New RS and Comments

File: ExchangeWebServices/Source/MS-OXWSCORE/TestSuite/MS-OXWSCORE_RequirementSpecification.xml [hunk 8]
```diff
@@ -3107,7 +3123,7 @@
 	<ns1:Requirement>
 		<ns1:REQ_ID>MS-OXWSCORE_R2069</ns1:REQ_ID>
 		<ns1:Doc_Sect>2.2.4.24</ns1:Doc_Sect>
-		<ns1:Description>[In t:ItemType Complex Type] [The element "PendingSocialActivityTagIds" with type "t:ArrayOfStringsType"] specifies tag ids to track draft social activity tags.</ns1:Description>
+		<ns1:Description>[In t:ItemType Complex Type] [The element "PendingSocialActivityTagIds" with type "t:ArrayOfStringsType"] specifies tag ids to track draft social activity tags.&lt;85&gt;</ns1:Description>
 		<ns1:Behavior>Protocol</ns1:Behavior>
 		<ns1:Scope>Both</ns1:Scope>
 		<ns1:IsNormative>Normative</ns1:IsNormative>
```
Comment (FangfangFan) [inline ExchangeWebServices/Source/MS-OXWSCORE/TestSuite/MS-OXWSCORE_RequirementSpecification.xml:3129]: it should be derived RS to verify like MS-OXWSCORE_R1730. Please add New RS and Comments

File: ExchangeWebServices/Source/MS-OXWSCORE/TestSuite/MS-OXWSCORE_RequirementSpecification.xml [hunk 9]
```diff
@@ -3116,7 +3132,7 @@
 	<ns1:Requirement>
 		<ns1:REQ_ID>MS-OXWSCORE_R1061</ns1:REQ_ID>
 		<ns1:Doc_Sect>2.2.4.9</ns1:Doc_Sect>
-		<ns1:Description>[In t:ManagementRoleType Complex Type] The ManagementRoleType complex type specifies the roles that are associated with a user or application.&lt;76&gt;
```
Comment (FangfangFan) [inline ExchangeWebServices/Source/MS-OXWSCORE/TestSuite/MS-OXWSCORE_RequirementSpecification.xml:3119]: it should be derived RS to verify like MS-OXWSCORE_R1730. Please add New RS and Comments

File: ExchangeWebServices/Source/MS-OXWSCORE/TestSuite/MS-OXWSCORE_RequirementSpecification.xml [hunk 10]
```diff
@@ -3116,7 +3132,7 @@
 	<ns1:Requirement>
 		<ns1:REQ_ID>MS-OXWSCORE_R1061</ns1:REQ_ID>
 		<ns1:Doc_Sect>2.2.4.9</ns1:Doc_Sect>
```
Comment (FangfangFan) [inline ExchangeWebServices/Source/MS-OXWSCORE/TestSuite/MS-OXWSCORE_RequirementSpecification.xml:3134]: Incorrect section number. It should be 2.2.4.26

Comment (FangfangFan) [reply ExchangeWebServices/Source/MS-OXWSCORE/TestSuite/MS-OXWSCORE_RequirementSpecification.xml:3134]: Incorrect section number issue with below RS:
MS-OXWSCORE_R1359
MS-OXWSCORE_R1360
MS-OXWSCORE_R1361
MS-OXWSCORE_R1364

File: ExchangeWebServices/Source/MS-OXWSCORE/TestSuite/MS-OXWSCORE_RequirementSpecification.xml [hunk 11]
```diff
@@ -5437,7 +5453,7 @@
 	<ns1:Requirement>
 		<ns1:REQ_ID>MS-OXWSCORE_R1087</ns1:REQ_ID>
 		<ns1:Doc_Sect>2.2.4.27</ns1:Doc_Sect>
-		<ns1:Description>[In t:MeetingSuggestionType Complex Type] Location: An element of type string, as defined in [XMLSCHEMA2] section 3.2.1, that represents the location of the meeting.</ns1:Description>
+		<ns1:Description>[In t:MeetingSuggestionType Complex Type] Location: An element of type string, as defined in [XMLSCHEMA2] section 3.2.1, that represents the location of the meeting.&lt;88&gt;</ns1:Description>
 		<ns1:Behavior>Protocol</ns1:Behavior>
```
Comment (FangfangFan) [inline ExchangeWebServices/Source/MS-OXWSCORE/TestSuite/MS-OXWSCORE_RequirementSpecification.xml:5457]: It should be derived RS to verify like MS-OXWSCORE_R1730. Please add New RS and Comments.
Derived RS should be verified in Test Case.

File: ExchangeWebServices/Source/MS-OXWSCORE/TestSuite/MS-OXWSCORE_RequirementSpecification.xml [hunk 12]
```diff
@@ -3941,7 +3957,7 @@
 	<ns1:Requirement>
 		<ns1:REQ_ID>MS-OXWSCORE_R1065</ns1:REQ_ID>
```
Comment (FangfangFan) [inline ExchangeWebServices/Source/MS-OXWSCORE/TestSuite/MS-OXWSCORE_RequirementSpecification.xml:3958]: It should be derived RS to verify like MS-OXWSCORE_R1730. Please add New RS and Comments.

File: ExchangeWebServices/Source/MS-OXWSCORE/TestSuite/MS-OXWSCORE_RequirementSpecification.xml [hunk 13]
```diff
@@ -3941,7 +3957,7 @@
 	<ns1:Requirement>
 		<ns1:REQ_ID>MS-OXWSCORE_R1065</ns1:REQ_ID>
 		<ns1:Doc_Sect>2.2.4.31</ns1:Doc_Sect>
-		<ns1:Description>[In t:NonEmptyArrayOfBaseItemIdsType Complex Type] [The element "RecurringMasterItemIdRanges" with type "RecurringMasterItemIdRangesType (section 2.2.4.18)"] Specifies a union of all occurrences of a recurring master, which fit into at least one of the specified ranges.&lt;80&gt;</ns1:Description>
```
Comment (FangfangFan) [inline ExchangeWebServices/Source/MS-OXWSCORE/TestSuite/MS-OXWSCORE_RequirementSpecification.xml:3944]: It should be derived RS to verify like MS-OXWSCORE_R1730. Please add New RS and Comments.

File: ExchangeWebServices/Source/MS-OXWSCORE/TestSuite/MS-OXWSCORE_RequirementSpecification.xml [hunk 14]
```diff
@@ -6320,15 +6336,16 @@
 	<ns1:Requirement>
 		<ns1:REQ_ID>MS-OXWSCORE_R2142</ns1:REQ_ID>
 		<ns1:Doc_Sect>2.2.4.43</ns1:Doc_Sect>
```
Comment (FangfangFan) [inline ExchangeWebServices/Source/MS-OXWSCORE/TestSuite/MS-OXWSCORE_RequirementSpecification.xml:6338]: It should be derived RS to verify like MS-OXWSCORE_R1730. Please add New RS and Comments.

Comment (FangfangFan) [reply ExchangeWebServices/Source/MS-OXWSCORE/TestSuite/MS-OXWSCORE_RequirementSpecification.xml:6338]: Adapter

**Review by v-yahan — COMMENTED** [review]
Please update the following RS in MS-OXWSFOLD:
R1041: Normative and it derived by <7>
R1053: same as R1041
R192: no need to udpate
R199, R444, R441: update reference section number
R1531, R56612: Missing Exchange 2019
R5663: need update

**Review by v-yahan — APPROVED** [review]

**Pull request merged**